canonical-web-and-design/maas.io
# Add image template

## Done
- Replaced lazyloading images with the [image template](https://github.com/canonical-web-and-design/canonicalwebteam.image-template)
- Removed lazyloading completely to top row of tour page because images are above the fold
- Removed unused classes on takeunder icons at the bottom of the tour page

## QA
- Go to [homepage](https://maas-io-canonical-web-and-design-pr-398.run.demo.haus/) and make sure all images load and match [https://maas.io](https://maas.io)
- Go to [/install](https://maas-io-canonical-web-and-design-pr-398.run.demo.haus/install) and make sure all images load and match [https://maas.io/install](https://maas.io/install)
- Go to [/how-it-works](https://maas-io-canonical-web-and-design-pr-398.run.demo.haus/how-it-works) and make sure all images load and match [https://maas.io/how-it-works](https://maas.io/how-it-works)
- Go to [/tour](https://maas-io-canonical-web-and-design-pr-398.run.demo.haus/tour) and make sure all images load and match [https://maas.io/tour](https://maas.io/tour)

## Issue / Card
Fixes #397
